p2p/src/net/types/mod.rs

```diff
@@ -60,6 +60,13 @@ pub struct PeerInfo<P> {
     pub subscriptions: BTreeSet<PubSubTopic>,
 }
 
+impl<P> PeerInfo<P> {
+    pub fn is_compatible(&self, network: [u8; 4]) -> bool {
```

Comment:
The `is_compatible` function should take a config, not a network, where multiple things can be tested.

Reply:
Sure, updated.